pkg/mimedb/db_test.go

```diff
@@ -29,3 +29,11 @@ func TestMimeLookup(t *testing.T) {
 		t.Fatalf("Invalid content type are found expected \"false\", got %t", compressible)
 	}
 }
+
+func TestTypeByExtension(t *testing.T) {
+	// Test TypeByExtension.
+	contentType := TypeByExtension(".txt")
+	if contentType != "text/plain" {
+		t.Fatalf("Invalid content type are found expected \"text/plain\", got %s", contentType)
```

Comment:
one more test needed for extensions which do not exist

Reply:
Done